apps/nrm/behaviours/resume-form-session.js

```diff
@@ -10,6 +10,21 @@ const _ = require('lodash');
 const encodeEmail = email => Buffer.from(email).toString('hex');
 
 module.exports = superclass => class extends superclass {
+  cleanSession(req) {
+    let cleanList = Object.keys(req.sessionModel.attributes);
+    const keepList = ['user-email', 'user-organisation', 'csrf-secret'];
```

Comment:
I thought we want to remove this csrf-secret?

Reply:
see me answer to the same question above